pkg/cookies/csrf.go

```diff
@@ -188,7 +226,9 @@ func decodeCSRFCookie(cookie *http.Cookie, opts *options.Cookie) (*csrf, error)
 	}
 
 	// Valid cookie, Unmarshal the CSRF
-	csrf := &csrf{cookieOpts: opts}
+	clock := clock.Clock{}
+	clock.Set(t)
+	csrf := &csrf{cookieOpts: opts, time: clock}
```

Comment:
This clock.Clock used to be only used for tests I think so I'm not sure what the impact of setting it here is?